frontend/src/pages/Graph/GraphToolbar/GraphSettings.tsx

```diff
@@ -81,12 +81,13 @@ type GraphSettingsProps = ReduxStateProps &
 type GraphSettingsState = { crippledFeatures?: KialiCrippledFeatures; isOpen: boolean };
 
 interface DisplayOptionType {
+  iconColor?: string;
   id: string;
   isChecked: boolean;
   isDisabled?: boolean;
   labelText: string;
   onChange?: () => void;
-  style?: string;
+  iconClassName?: string;
```

Comment:
Just a minor thing, that should be alphabetized also?

Reply:
yes.